tests/e2e/runner/runner.go

```diff
@@ -46,12 +47,25 @@ func GetClient() client.Client {
 func InitializeRunner(execPath_ string, grpcEp string, networkRunnerLogLevel string) error {
 	execPath = execPath_
 
-	var err error
+	// Create the logger
+	logLevel, err := logging.ToLevel(networkRunnerLogLevel)
+	if err != nil {
+		return err
+	}
+
+	logFactory := logging.NewFactory(logging.Config{
+		DisplayLevel: logLevel,
+		LogLevel:     logging.Off,
```

Comment:
can we add a comment why should this be `logging.Off`? does not seem intuitive

Reply:
Added

Reply:
just want the logs on stdout, and want to prevent the client logger to create a log file, which is not needed in CI, and also usually not needed on local executions